src/main/java/name/webdizz/sonar/grammar/spellcheck/GrammarDictionaryLoader.java

```diff
@@ -80,6 +71,19 @@ private SpellDictionary loadSpellDictionary(final Reader dictionaryReader, final
         }
         return dictionary.get();
     }
+    private Properties readProperties(String configFileName) {

```

Comment:
Class is responsible for loading of dictionary, so let's extract properties loading into dedicated class